modules/processing/memory.py

```diff
@@ -929,9 +932,25 @@ def __init__(self, memfile, osprofile=None):
             if pid:
                 self.mask_pid.append(int(pid))
 
+        response = {}
+
+        task = db.view_task(db.list_tasks(limit=1)[0].id, details=True)
```

Comment:
What is happening here; getting an instance of the current task? If so we can just replace this with `self.task`.